src/axom/core/execution/synchronize.hpp

```diff
@@ -6,33 +6,30 @@
 #ifndef AXOM_CORE_EXECUTION_SYNCHRONIZE_HPP_
 #define AXOM_CORE_EXECUTION_SYNCHRONIZE_HPP_
 
-#include "axom/config.hpp"                          /* for compile time defs. */
-#include "axom/core/Macros.hpp"                     /* for AXOM_STATIC_ASSERT */
-#include "axom/core/execution/execution_space.hpp"  /* execution_space traits */
+#include "axom/config.hpp"                         /* for compile time defs. */
+#include "axom/core/Macros.hpp"                    /* for AXOM_STATIC_ASSERT */
+#include "axom/core/execution/execution_space.hpp" /* execution_space traits */
 
 namespace axom
 {
-
 /*!
  * \brief Synchronizes all execution threads when using an ASYNC policy with
  *  the specified execution space.
  *
  * \tparam ExecSpace the execution space
  */
-template < typename ExecSpace >
-inline void synchronize() noexcept
+template <typename ExecSpace> inline void synchronize() noexcept
 {
-  AXOM_STATIC_ASSERT( execution_space< ExecSpace >::valid() );
+  AXOM_STATIC_ASSERT(execution_space<ExecSpace>::valid());
 
 #ifdef AXOM_USE_RAJA
-  using sync_policy = typename execution_space< ExecSpace >::sync_policy;
-  RAJA::synchronize< sync_policy >( );
+  using sync_policy = typename execution_space<ExecSpace>::sync_policy;
+  RAJA::synchronize<sync_policy>();
 #endif
 }
 
-template < >
-inline void synchronize< SEQ_EXEC >() noexcept { }
+template <> inline void synchronize<SEQ_EXEC>() noexcept { }
 
-}
+}  // namespace axom
```

Comment:
Note: `clang-format` adds comments at the end of namespace.  It also fixes incorrect comments for these!